k8s.tf

```diff
@@ -198,8 +200,13 @@ resource "kubernetes_service" "waggle_dance" {
       port        = local.wd_port
       target_port = local.wd_port
     }
-    type                        = "LoadBalancer"
-    load_balancer_source_ranges = var.ingress_cidr
+    type                              = "LoadBalancer"
+    load_balancer_source_ranges       = var.ingress_cidr
+    external_traffic_policy           = var.k8s_svc_spec.external_traffic_policy
```

Comment:
What happens if `var.k8s_svc_spec.external_traffic_policy` is not specified? It is not gonna set some attributes to null? Is it that okay?

Reply:
if not specified, this means passing "null".
https://developer.hashicorp.com/terraform/language/expressions/types#null

Reply:
If you set it to null, is it gonna take the default value?

Reply:
yeah that's what the docs say